From 195b5d0725b7caf016def2ecedb915c1422e043a Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Tue, 1 Sep 2015 17:21:53 +0200
Subject: [PATCH] cms/cmsifup.sh: do not use ifup -m

we don't want to bring the interface up several times

(cherry picked from commit 0852c38b7a2b893e4f9e5894ba09eebc649f047f)
---
 modules.d/80cms/cmsifup.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/80cms/cmsifup.sh b/modules.d/80cms/cmsifup.sh
index b1b9616..902df8d 100755
--- a/modules.d/80cms/cmsifup.sh
+++ b/modules.d/80cms/cmsifup.sh
@@ -35,4 +35,4 @@ fi
 IFACES="$IFACES $DEVICE"
 echo "$IFACES" >> /tmp/net.ifaces
 
-ifup $DEVICE -m
+exec ifup "$DEVICE"
